impl/src/java/org/sakaiproject/attendance/logic/AttendanceLogicImpl.java

```diff
@@ -295,7 +297,8 @@ public void updateAttendanceRecordsForEvent(AttendanceEvent aE, Status s, String
 		}
 
 		dao.updateAttendanceItemStats(itemStats);
-		dao.updateAttendanceRecords(recordsToUpdate);
+
+		//dao.updateAttendanceRecords(recordsToUpdate);
```

Comment:
Can you add some context why this line is being removed? If is save to delete, just delete it

Reply:
Sure. UpdateAttendanceRecords loops through a list of records passed into it and calls the Hibernate saveOrUpdate method on each one. Instead, I just had it do updateAttendaceRecord [singular] on each of the changed records in an earlier loop [at 277]. This doesn't necessarily reduce the number of Hibernate calls or anything, but it cuts out a redundant loop.
Sorry I forgot to mention that before.